tests/test_convert.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python
```

Comment:
Since this file is full of print calls, it should probably have `from __future__ import print_function` (unless it is intended to Python 3 only).